scss/_patterns_tabs.scss

```diff
@@ -93,4 +93,37 @@
       }
     }
   }
+
+  .p-tabs-segmented {
+    &.is-dense .p-tabs-segmented__button {
```

Comment:
Every component should push things that come after it by the same amount. Also, to ensure child elements' margins are inside the component, we need overflow: auto;
```    margin-bottom: $spv-outer--scaleable;
         overflow: auto;
```

Reply:
Ah, we were following `is-dense` button modifier, which reduces the bottom margin.

Reply:
@lyubomir-popov I reverted the `overflow: auto` and `align-items: flex-start` changes, I noticed they broke the pattern at smaller widths:
![buttons](https://user-images.githubusercontent.com/2376968/129368556-27c7b2ab-6ea5-42dc-b057-c3b2983a2b7e.gif)